WalletWasabi/Blockchain/Transactions/TransactionProcessor.cs

```diff
@@ -9,6 +9,32 @@
 
 namespace WalletWasabi.Blockchain.Transactions
 {
+	public class DoubleSpendReceivedEventArgs
```

Comment:
Maybe `DoubleSpendReceivedEventArgs` & `ReplaceTransactionReceivedEventArgs` classes should be placed under `TransactionProcessor` class.

Reply:
I think they should be moved to a new file in the same folder.

Reply:
Ack